src/docs/sdk/unified-api/tracing.mdx

```diff
@@ -70,6 +70,8 @@ tree as well as the unit of reporting to Sentry.
     - When a `Span` is created, set the `startTimestamp` to the current time
     - `SpanContext` is the attribute collection for a `Span` (Can be an implementation detail)
     - The relation between parent - child is captured in the property `parentSpanId`
+    - `Span` should have a method called `toTraceparent` which returns a string `sentry-trace` that could be sent as a header
+    - Similar `SpanContext` should have a static method called `fromTraceparent` which prefills a `SpanContext` with data received from a `sentry-trace` string
```

Comment:
- `SpanContext` is an implementation detail, as indicated above, so I'd not define required methods on it. `from/toTraceparent` should be in `Span` and `Transaction`. JS defines both methods on `Span` and they are inherited by `Transaction`.
- We need to document the format of the string used for the `sentry-trace` header.

Reply:
I would say `SpanContext` while implementation detail we can come with an opinion to have this there.